compaction/compaction.cc

```diff
@@ -1268,8 +1314,8 @@ class cleanup_compaction final : public regular_compaction {
     }
 
 public:
-    cleanup_compaction(table_state& table_s, compaction_descriptor descriptor, compaction_data& cdata)
-        : regular_compaction(table_s, std::move(descriptor), cdata)
+    cleanup_compaction(table_state& table_s, compaction_descriptor descriptor, compaction_data& cdata, compaction_read_monitor_generator_wrapper& monitor_generator)
+        : regular_compaction(table_s, std::move(descriptor), cdata, monitor_generator, true)
     {
```

Comment:
we can just pick the regular compaction's default. but it's okay to make it explicit too.